packages/client/components/JiraScopingSelectAllIssues.tsx

```diff
@@ -24,28 +25,18 @@ interface Props {
   meetingId: string
   issues: JiraScopingSelectAllIssues_issues
   usedJiraIssueIds: Set<string>
-
 }
 
 const JiraScopingSelectAllIssues = (props: Props) => {
   const {meetingId, usedJiraIssueIds, issues} = props
+  const issueIds = issues.map(issueEdge => issueEdge.node.id)
   const atmosphere = useAtmosphere()
   const {onCompleted, onError, submitMutation, submitting} = useMutationProps()
-  const unusedIssues = useMemo(() => {
-    const unusedIssues = [] as string[]
-    issues.forEach(({node}) => {
-      if (!usedJiraIssueIds.has(node.id)) {
-        unusedIssues.push(node.id)
-      }
-    })
-    return unusedIssues
-  }, [usedJiraIssueIds, issues])
-  const selectAll = unusedIssues.length === 0 ? true : unusedIssues.length === issues.length ? false : null
-
+  const [unusedIssues, selectAll] = useUnusedRecords(issues, usedJiraIssueIds)
   const onClick = () => {
     if (submitting) return
     submitMutation()
-    const updateArr = selectAll === true ? Array.from(usedJiraIssueIds) : unusedIssues
+    const updateArr = selectAll === true ? issueIds : unusedIssues
```

Comment:
@mattkrick i changed this because clicking "Select all" when all items are already checked was deselecting all possible issues instead of just the ones that are currently shown.

I thought this behavior was unintuitive and unintentional. Here's a loom video to show what the problem was: https://www.loom.com/share/e240af2538b54a22a281816c2f586455